src/BusinessNotificationCenter.jsx

```diff
@@ -0,0 +1,120 @@
+import './App.css';
+import { useEffect, useState } from 'react';
+import { useBackend } from './contexts/BackendContext';
+import { SearchIcon, PlusSquareIcon, BellIcon, HamburgerIcon } from '@chakra-ui/icons';
+import {
+  Text,
+  InputGroup,
+  InputLeftElement,
+  Icon,
+  Menu,
+  MenuButton,
+  Button,
+  Input,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from '@chakra-ui/react';
+import {
+  Modal,
+  ModalOverlay,
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'ModalOverlay' is defined but never used.

Reply:
remove unused variables